quisby/benchmarks/passmark/graph.py

```diff
@@ -0,0 +1,124 @@
+
```

Comment:
We need to work on getting the generic results generation in, will save a lot of work and duplicate code.

Reply:
True that :(